nssp_demo/create_hubverse_table.R

```diff
@@ -0,0 +1,150 @@
+draws_to_quantiles <- function(forecast_dir,
+                               report_date) {
+  message(glue::glue("Processing {forecast_dir}..."))
+  draws_path <- fs::path(forecast_dir,
+    "forecast_samples",
+    ext = "parquet"
+  )
+  location <- fs::path_file(forecast_dir)
+
+  draws <- arrow::read_parquet(draws_path) |>
+    dplyr::filter(date >= lubridate::ymd(report_date) - lubridate::days(8))
```

Comment:
Can you explain this line? and put it in a comment?

Reply:
Done in 5ce3003